app/models/post.rb

```diff
@@ -10,10 +10,15 @@ class Post < ApplicationRecord
 
   after_create do
     ActionCable.server.broadcast "posts_realtime", { row: PostsController.render(locals: {post: Post.last}, partial: 'post').html_safe }
+    ActionCable.server.broadcast "topbar", { review: Post.feedbacks_count }
   end
 
   after_create :autoflag
 
+  def self.feedbacks_count
```

Comment:
Could be renamed to something more descriptive - I see two options: 

 1. Rename to `review_count` or similar (easy), or
 2. Add a `without_feedback` [scope](http://guides.rubyonrails.org/active_record_querying.html#scopes), then calling `Post.without_feedback.count` would work. (harder)